examples/1d_stencil/CMakeLists.txt

```diff
@@ -57,31 +57,20 @@ foreach(example_program ${example_programs})
   source_group("Source Files" FILES ${sources})
 
   # add example executable
-  add_hpx_executable(${example_program}
+  add_hpx_executable(${example_program}_example
```

Comment:
The `_example` suffix is the only thing I'm unsure about here. I know that for consistency with tests (which get a `_test` suffix) it would make sense to add `_example`, but then we end up with things like `./bin/hello_world_distributed_example` which to my eye looks ugly. I'd prefer to leave things as they are for now.

Reply:
Ok sure!